flit.ini

```diff
@@ -0,0 +1,38 @@
+[metadata]
+module = ipfsapi
+
+author       = py-ipfs-api team
+author-email = 
+home-page    = https://github.com/ipfs/py-ipfs-api
+keywords     = ipfs storage distribution development
+license      = MIT License
+
+description-file = README.md
+
+# Unfortunately these currently need to be duplicated from `requirements.txt`
```

Comment:
I wish we could only include these in one place and simply parse them out of that one place in every other context, but I don't see how to do it with .ini files in the mix. Sigh...